Criado titulo, entidades entidadesBase Repositorio

LucasPerezAlves opened the pull request:
-Foi adicionado o titulo "Fox Budgets na tela do HTML"
-Criado as entidades para o administrador
-Criado as entidadesBase para o administrador
-Criado o Repositorio, porém comentado, pois precisa de outros componentes ainda não criados
então foi comentado

LucasPerezAlves committed:
Criado titulo, entidades entidadesBase Repositorio
-Foi adicionado o titulo "Fox Budgets na tela do HTML"
-Criado as entidades para o administrador
-Criado as entidadesBase para o administrador
-Criado o Repositorio, porém comentado, pois precisa de outros componentes ainda não criados
então foi comentado

GustavoPrim commented on Repositorio/Entidades/EntidadeBase.cs:
```diff
@@ -0,0 +1,9 @@
+    {
+        public int Id { get; set; }
+        public string Nome { get; set; }
+
```
Remover linha em branco

GuilhermeDMueller commented on Repositorio/Repositorios/AdministradorRepositorio.cs:
```diff
@@ -0,0 +1,42 @@
+
+namespace Repositorio.Repositorios
+{
+    internal class AdministradorRepositorio : IAdministradorRepositorio
```
Trocar de internal para public

GustavoPrim committed:
Merge branch 'develop' into feature/adicionar-administrador

LucasPerezAlves committed:
Implementado AdministradorRepositorio,comentarios
-Foi feito a alteração que foi solicitada nos comentários
-Implementado o que estava faltando no AdministradorRepositorio

LucasPerezAlves committed:
Merge branch 'feature/adicionar-administrador' of https://github.com/GustavoPrim/Entra21-TrabalhoFinal-FoxBudgets into feature/adicionar-administrador

GuilhermeDMueller reviewed (commented):
Trocar a class AdministradorRepositorio de Internal para public
